docs/content/en/docs/how-tos/filesync/_index.md

```diff
@@ -10,40 +10,38 @@ This page discusses how to set up file sync for files that don't require full re
 File sync is alpha and may change between releases.
 {{< /alert >}}
 
-Skaffold supports copying changed files to a deployed containers so as to avoid the need to
-rebuild, redeploy, and restart the corresponding pod.  The file copying is enabled
-by adding a `sync` section with _sync rules_ to the `artifact` in the `skaffold.yaml`.
-
-The following example will cause any changes to JavaScript files under the _context_ directory
-to be copied to the deployed container into the container's `WORKDIR`.
-
-```yaml
-apiVersion: skaffold/v1beta8
-kind: Config
-build:
-  artifacts:
-  - image: gcr.io/k8s-skaffold/node-example
-    context: node
-    sync:
-      '.filebaserc': .
-      '*.html': static
-      '**/*.png': assets
-      '***/*.md': content
-```
-A double-asterisk (`**/`) applies recursively to all subdirectories but flattens the result,
-stripping the subdirectory structure.
-A triple-asterisk (`***/`) applies recursively to all subdirectories but retains
-the subdirectory structure.  
-
-Under the hood, Skaffold monitors and creates a tar file with changed files that match
-the sync rules.  This tar file is sent and extracted on the corresponding containers. 
-
-### Limitations
+Skaffold supports copying changed files to a deployed container so as to avoid the need to rebuild, redeploy, and restart the corresponding pod.
+The file copying is enabled by adding a `sync` section with _sync rules_ to the `artifact` in the `skaffold.yaml`.
+Under the hood, Skaffold creates a tar file with changed files that match the sync rules.
+This tar file is sent to and extracted on the corresponding containers. 
+
+### Manual sync mode
+
+A manual sync rule must specify the `src` and `dest` field.
+The `src` field is a glob pattern to match files relative to the artifact _context_ directory, which may contain `**` to match nested files.
+The `dest` field is the destination location in the container.
+It may be absolute or relative, in which case files are put below the container's `WORKDIR`.
```

Comment:
Just for clarity, I would rephrase this as `Destination paths may be absolute or relative. If the destination is a relative path, an absolute path will be inferred by prepending the path with the container's WORKDIR`